src/include/miopen/any_solver.hpp

```diff
@@ -263,6 +263,17 @@ struct AnySolver
                 }
                 MIOPEN_LOG_I2("Perf Db: Invalid Config: " << value.SolverDbId());
             }
+            else if(!value.AltSolverDbId().empty() && db.Load(ctx, value.AltSolverDbId(), config))
+            {
+                MIOPEN_LOG_I("Perf Db: alternate record loaded: " << value.AltSolverDbId());
+                if(value.IsValidPerformanceConfig(ctx, config))
+                {
+                    return config.ToString();
+                }
+                MIOPEN_LOG_WE("Invalid alternate record loaded from Perf Db: "
+                              << value.AltSolverDbId() << ": " << config
+                              << ". Performance may degrade.");
```

Comment:
I think `"Perf Db:"` prefix should be added here for consistency.
Do you need to print config here? It is not printed for the first loading from Db. Also do we need to print warning about performance in this case?

Reply:
I have added the prefix and reduced the log level, printing the config can be beneficial for debugging.